### ISSUE TITLE
User profile is difficult to find

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#15493]
We've received a few separate internal reports that the profile icon is hard to find, and @mattmiklic proposed in a comment on Dec 7th that we might be able to make the profile icon visible on all top-level screens. 

<sup>(internal reference: p4a5px-2FS-p2#comment-11594)</sup>

Apr 8, 2020

> I'm testing a WPiOS PR, and I'm just unable to figure out where to logout. Me tab is gone and inaccessible.
> 
> <sup>(internal reference: p1586351671189200-slack-apple-dev on Apr 8, 2020)</sup>

Dec 4, 2020

> I was curious to find my profile to see what account I was logged in and all that and it took me surprisingly long time to find it.
> 
> <sup>(internal reference: p4a5px-2FS-p2 on Dec 4, 2020, also see https://github.com/wordpress-mobile/WordPress-iOS/issues/15001#issuecomment-739147880)</sup>

Dec 5, 2020

> I can really relate to this. Being an iOS first user, ever since the ‘Me’ section was moved from the bottom footer menu icons, I’m always stumbling around to find the Profile Icon and I use the app every day. I find it especially confusing when I’m on the ‘My Sites’ list and don’t have a site selected.
> 
> <sup>(internal reference: p4a5px-2FS-p2#comment-11585 on Dec 5, 2020)</sup>

**Comment by designsimply** [comment]
Adding a personal note, I was also frustrated when the profile icon was moved but I did not report it as a bug because I feel I have a special case where I believe I use the account screen FAR MORE than others. I also think that when items are moved that users of any system can become frustrated—even if the move is a good one, and I considered that. As for me, I am constantly checking the version number, app settings, and logs as part of beta testing so I feel a pain in not having it top-level in a way I think others don't. Still, since it has been reported a few times internally, I think we should consider a change even if we haven't heard this particular feedback bubble up from users because I think, as something opinionated, it's a type of feedback that can get lost in the din. We don't _have to_ make a change! But I would love to see us consider our internal feedback in any case.
